applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.c

```diff
@@ -0,0 +1,206 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "bt_mgmt.h"
+
+#include <zephyr/zbus/zbus.h>
+#include <zephyr/bluetooth/bluetooth.h>
+#include <zephyr/bluetooth/conn.h>
+#include <zephyr/bluetooth/hci.h>
+
+#include "macros_common.h"
+#include "nrf5340_audio_common.h"
+#include "ble_audio_services.h"
+#include "ble_hci_vsc.h"
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(bt_mgmt, CONFIG_BT_MGMT_LOG_LEVEL);
+
+ZBUS_CHAN_DEFINE(bt_mgmt_chan, struct bt_mgmt_msg, NULL, NULL, ZBUS_OBSERVERS_EMPTY,
+		 ZBUS_MSG_INIT(0));
+
+static void num_conn_get(struct bt_conn *conn, void *data)
+{
+	int ret;
+	uint8_t *num_conn = (uint8_t *)data;
+	struct bt_conn_info info;
+
+	ret = bt_conn_get_info(conn, &info);
+	if (ret) {
+		LOG_ERR("Failed to get conn info for %p: %d", (void *)conn, ret);
+		return;
+	}
+
+	if (info.state != BT_CONN_STATE_CONNECTED) {
+		return;
+	}
+
+	(*num_conn)++;
```

Comment:
I don't get this. We want to get the number of conns but they are incremented here.